README.md

```diff
@@ -8,39 +8,47 @@
 [![Ansible Role](https://img.shields.io/ansible/role/d/42773.svg?style=popout-square)](https://galaxy.ansible.com/arillso/restic)
 
 ## Description
+
 [Restic](https://github.com/restic/restic) is a versatile Go based backup
 solution which supports multiple backends, deduplication and incremental
 backups.
 
-This role installs restic on a client, configures the backup repositories
+This role installs Restic on a client, configures the backup repositories
 and optionally sets up cronjobs to run said backups.
 Aditionally, it will setup executable scripts to run a Backup manually.
 
 > This Project borrows heavily from the
 > [donat-b/ansible-restic](https://github.com/donat-b/ansible-restic)
-> ansible role. We try to make this role more semver deployment friendly
+> ansible role. We try to make this role more server deployment friendly
 > by allowing to use version tags and keep these snapshots and adapting the
 > automated backup definition for use with windows systems.
 
 ### Backup Scripts
-This role will create a backup script and a file with credentials usable with the `source` command on linux for each backup in the `restic_script_dir`.
+
+This role will create a backup script and a file with credentials usable with the `source` command on Linux for each backup in the `restic_script_dir`.
 These executable scripts can be used to manually trigger a backup action, but
 are also used for automated backups if you have set `restic_create_cron` to true.
-make sure to not change the files manually, as this can interfere with your
-backups quite a bit.
 
-on Linux, if you want to take a manual snapshot, you can run the backup like this:
+
+By default, this role will use the Jinja template file located at `templates/restic_script_Linux.j2` to construct the backup script. However, if you wish to customize it further, you may override the template, and set the`restic_backup_script_template` variable to a path to your own template file.
```

Comment:
It would be nice to have a direct link here, so people won't have to navigate the role files 😬